app/src/main/java/com/nervousfish/nervousfish/activities/CreateProfileHelper.java

```diff
@@ -119,7 +119,7 @@ private boolean isValidName(final String name) {
      * @param password The password that has been entered.
      * @return a {@link boolean} indicating whether or not the password is valid.
      */
-    private boolean isValidPassword(final String password) {
+    private static boolean isValidPassword(final String password) {
```

Comment:
Because validateName/password use member fields and isValidPassword not. Actually, I didn't change this myself, that would cost too much time to do in the last few weeks xD . Just clicked auto cleanup in Intellij and I liked the result